estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,44 @@
+from odoo import api, fields, models, SUPERUSER_ID
+from dateutil.relativedelta import relativedelta
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "estate property offer"
+
+    price = fields.Float()
+    status = fields.Selection(
+        selection = [
+            ('accepted','Accepted'),
+            ('refused','Refused'),
+        ],
+        copy=False,
+    )
+
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    validity = fields.Integer(default=7, string="Validity(days)", )
+    date_deadline = fields.Date(string="Deadline Date", compute="_compute_deadline", inverse="_inverse_deadline")
+
+    @api.depends("validity")
+    def _compute_deadline(self):
+        for record in self:
+            default_creation_date = record.create_date if record.create_date else fields.Date.today()
+            record.date_deadline = relativedelta(days=record.validity) + default_creation_date
+    
+    def _inverse_deadline(self):
+        for record in self:
+            default_creation_date = record.create_date if record.create_date else fields.Date.today()
```

Comment:
```suggestion
            default_creation_date = record.create_date or fields.Date.today()
```
Another way to do it